static/xref/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Search cross references</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="script.js" defer></script>
+  </head>
+  <body>
+    <form action="/xref" id="xref-search" name="xref-search" method="post">
+      <div class="field">
+        <label for="term">Terms, concept, or WebIDL</label>
+        <input name="term" placeholder="EventTarget, event target" />
+      </div>
+      <div class="field">
+        <label for="cite">Specification(s) short name</label>
+        <div class="selections" data-for="cite"></div>
+        <input name="cite" placeholder="html,webidl,payment-request" />
+      </div>
+      <div class="field">
+        <label for="types">Types</label>
+        <div class="selections" data-for="types"></div>
+        <input name="types" placeholder="dfn,interface,_IDL_" />
+      </div>
+      <div class="field">
+        <label for="for">For</label>
+        <input name="for" placeholder="ResizeObservation" />
+      </div>
+      <button>Search</button>
+    </form>
+    <section id="output"></section>
+    <template id="table">
+      <table>
+        <thead>
+          <tr>
+            <th>Spec</th>
+            <th>Type</th>
+            <th>for</th>
```

Comment:
I’m thinking we need a last row that shows how to cite. Like, if it’s IDL, {{WhatEver}}, if it’s a “concept for”, [= list/for each =], and so on. Then Editors just copy paste the right thing.